docs/conf.py

```diff
@@ -49,7 +49,7 @@ def _do_not_evaluate_in_jax(
 # -- Project information -----------------------------------------------------
 
 project = 'JAX'
-copyright = '2024, The JAX Authors. NumPy and SciPy documentation are copyright the respective authors.'
+copyright = '2024, The JAX Authors.'
```

Comment:
Do we still end up with a trailing double period?

Reply:
Fixed, thanks!